README.md

```diff
@@ -21,7 +21,7 @@ recommend using the [app skeleton](https://github.com/cakephp/app) as
 a starting point. For existing applications you can run the following:
 
 ``` bash
-$ composer require cakephp/cakephp:"~3.2"
+$ composer require cakephp/cakephp:"~3.3"
```

Comment:
^3.3.0 should
https://getcomposer.org/doc/articles/versions.md#caret

Not sure what ^3.0 does though.